src/pages/messages/TasksPage.tsx

```diff
@@ -172,10 +169,10 @@ const TasksPage = () => {
                     setTicketTypes({ ...ticketTypes, publishingRequest: !ticketTypes.publishingRequest });
                     resetPaginationAndNavigate(searchParams, navigate);
                   }}>
-                  <InsertDriveFileOutlinedIcon fontSize="small" />
-                  {ticketTypes.publishingRequest && publishingRequestCount
-                    ? `${t('my_page.messages.types.PublishingRequest')} (${publishingRequestCount})`
-                    : t('my_page.messages.types.PublishingRequest')}
+                  <TicketTypeTag
+                    count={ticketTypes.publishingRequest && publishingRequestCount ? publishingRequestCount : undefined}
+                    type={TicketTypeEnum.PublishingRequest}
+                  />
```

Comment:
New component that gives us this:

<img width="348" height="254" alt="image" src="https://github.com/user-attachments/assets/6f53569c-05eb-45a2-ad96-d4be10177d0c" />